src/Components/bookHub/BookCard.js

```diff
@@ -50,31 +50,31 @@ export default function BookCard({
   useEffect(getAuthor, []);
 
   return ( // horizontal scroll not implemented
-    <Card className="card" sx={{ maxWidth: 345 }}>
+    <div className="card" style={{ margin: inCarousel ? '0' : '30px 16px 10px 16px' }}>
       <Link class="link" to={`/book/${id}`}>
-        <CardActionArea className="cardActionArea">
-          <div className="cover">
-            <CardMedia
+        <div className="cardActionArea">
+          <div className="img-container">
+            <img
               className="image"
-              component="img"
-              image={image}
+              // component="img"
+              src={image}
               alt="missing_book_cover"
             />
           </div>
 
-          <Typography className={classes.title}>
+          <p className="book-card-text">
             {title}
-          </Typography>
+          </p>
           <Link class="link" to={`/creator/${author}`}>
```

Comment:
can you also reorganize this so the link to the author isn't nested in the link to the book?

Reply:
may require some restyling